submissions/HannaSyn/Popup/css/style.css

```diff
@@ -0,0 +1,240 @@
+/*reset*/
+@import url("https://fonts.googleapis.com/css2?family=Titillium+Web:wght@700&display=swap");
+* {
+  padding: 0;
+  margin: 0;
+  border: 0;
+}
+
+*, *:before, *:after {
+  box-sizing: border-box;
+}
+
+nav, footer, header, aside {
+  display: block;
+}
+
+html, body {
+  height: 100%;
+  width: 100%;
+  font-size: 100%;
+  line-height: 1;
+  font-size: 14px;
+  -ms-text-size-adjust: 100%;
+  -moz-text-size-adjust: 100%;
+  -webkit-text-size-adjust: 100%;
+  overflow: hidden;
+}
+
+input, button, textarea {
+  font-family: inherit;
+}
+
+input::-ms-clear {
+  display: none;
+}
+
+button {
+  cursor: pointer;
+}
+
+button::-moz-focus-inner {
+  padding: 0;
+  border: 0;
+}
+
+a, a:visited {
+  text-decoration: none;
+}
+
+a:hover {
+  text-decoration: none;
+}
+
+ul li {
+  list-style: none;
+}
+
+img {
+  vertical-align: top;
+  max-width: 100%;
+}
+
+h1, h2, h3, h4, h5, h6 {
+  font-size: inherit;
+  font-weight: inherit;
+}
+
+/*--------------------*/
+/*fonts*/
+/*--------------------*/
+.header {
+  position: absolute;
+  width: 100vw;
+  height: 80px;
+  font-family: "Titillium Web", sans-serif;
+  background: #94dbf745;
+}
+
+.navigation {
+  max-width: 1100px;
+  margin: 25px auto 0;
+}
+.navigation__list {
+  display: flex;
+  justify-content: flex-end;
+}
+.navigation__link {
+  flex: 0 0 30px;
+  margin: 0 30px 0 0;
+  width: 30px;
+  height: 30px;
+}
+.navigation__link img {
+  object-fit: contain;
+  width: 100%;
+  height: 100%;
+}
+
+.menu {
+  position: relative;
+}
+.menu__input {
+  position: absolute;
+  width: 30px;
+  height: 30px;
+  cursor: pointer;
+  opacity: 0;
+}
+.menu__label {
+  position: relative;
+  cursor: pointer;
+}
+.menu__label::before {
```

Comment:
This pointer/triangle logically better to create as pseudoelement of popup not label.

Reply:
It wasn't working with popup I guess because it's out of this block

Reply:
I mean if I take this popup component and paste him into a new place will this pointer/triangle present at the new place?

Reply:
Fixed.